src/streams/StreamScreen.js

```diff
@@ -1,12 +1,12 @@
 /* @flow strict-local */
 import React, { PureComponent } from 'react';
-import { View } from 'react-native';
+import { View, TouchableNativeFeedback, Platform, Clipboard, ToastAndroid } from 'react-native';
```

Comment:
I like React Native's naming of `ToastAndroid` better, since it's very clear that it only works on Android, and I see that you're looking for something that works on both platforms. You'll find it in src/utils/info.js, called `showToast`.

Take a look at some other places where `showToast` is used; you'll find that some (it should be all!) pass `_('some string literal')` instead of `'some string literal'`. The `_` ensures that the string displays in the user's chosen language. You'll need access to `_`, and I think the right way to get that is to put this at the top of the `StreamScreen` class:

```
  static contextType = TranslationContext;
  context: GetText;
```

This makes StreamScreen a consumer of the `TranslationContext`, which was created with `React.createContext` in the [new React Context API](https://reactjs.org/docs/context.html#legacy-api). It might be helpful to follow the example of the `LanguagePicker` component (src/settings/LanguagePicker.js), which does the same thing. The strategy is also described in the JSDoc on the `GetText` type in src/types.js.

Then, in `handleCopy`, declare `const _ = this.context;`. The naming of `_` is to make it as unobtrusive as possible, while still ensuring that it shows in its translated form.

Then, you should add an entry into static/translations/messages_en.json. That way, the English version will be available, and when we sync to Transifex, it will be available for translators to translate into other languages (there's a [doc](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/translations.md) about that, if you're interested). But you don't have to worry about that; the essential thing for this PR branch is that an entry gets added into messages_en.json.

Also, I'd like to be consistent with other user-facing strings, with capitalization and punctuation; what do you think of "Copied email address" (and maybe @gnprice, do you have thoughts on this)?

Reply:
Thanks for pointing me out to the right places! Will definitely take care of the `TouchableNativeFeedback` and the `AndroidToast`.  Thanks on the great explanation of the translation workflow, (was wondering why `_` was being used like that in codebase while quickly skimming through it.)
About the "Copied email address" - I agree, being consistent with other strings is better and maybe let's hear about it from @gnprice as well?